tests/unit/dojson/test_dojson_experiments.py

```diff
@@ -22,10 +22,12 @@
 
 from __future__ import absolute_import, division, print_function
 
+import pytest
+
 from dojson.contrib.marc21.utils import create_record
 
 from inspirehep.dojson.experiments import experiments
-from inspirehep.dojson.utils import strip_empty_values
+from inspirehep.dojson.utils import clean_record, clean_record
```

Comment:
Apparently moving and assembling IKEA furniture for a week doesn't count as vacation (I wish I could ping my girlfriend on GitHub).